### PR TITLE
Adding some classes to help with Integration Tests

### PR DESCRIPTION
**Pull request by fmacleal**
- Some refactors made on the Integration Tests classes
- Some classes to easily spin up an rskj node on the integration test, with possibility to check the output from it

## Motivation and Context

To have a better structure of classes to create integration tests faster and simpler with the possibility to easily start multiple nodes, necessary to validate some scenarios of sync and communication between nodes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Review by rmoreliovlabs — DISMISSED** [review]
LGTM

File: rskj-core/src/integrationTest/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+## rskj-core integrationTests
+
+This module contains integration tests for the RSKj node, these tests aim to validate core functionalities from the node
+and validate key features from the tool. We takes sometime to be executed and are part of the CI pipeline to ensure the
```
Comment (Vovchyk) [inline rskj-core/src/integrationTest/README.md:4]:
```suggestion
and validate key features from the tool. It takes sometime to be executed and is part of the CI pipeline to ensure the
```

File: rskj-core/src/integrationTest/README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+contribute with new tests, please feel free to open a PR. To make this task easy, some classes were created to help the
+creation of nodes inside integration tests scenarios and the validation that they worked as expected.
+
+### Running the integration tests
+
+You will need gradle installed in your machine to run the integration tests. To run the integration tests, execute the 
+following command:
+
+```shell
+./gradlew --no-daemon --stacktrace integrationTest
```
Comment (Vovchyk) [inline rskj-core/src/integrationTest/README.md:17]:
```suggestion
./gradlew --stacktrace integrationTest
```
with daemon next executions might run faster. Is there any specific reason for using this flag?

**Review by Vovchyk — CHANGES_REQUESTED** [review]

**Comment by fmacleal** [comment]
pipeline: run

**Comment by Vovchyk** [comment]
pipeline:run

**Review by Vovchyk — APPROVED** [review]

**Pull request merged**